pasta/base/annotate.py

```diff
@@ -1528,17 +1538,52 @@ def check_slice_includes_step(self, node):
       # but also treats a[::None] exactly the same.
       return self.tokens.peek_non_whitespace().src not in '],'
 
+    def create_indent_markers(self, ws, indent=None):
+      if indent is None:
+        indent = self._indent
+      # Replace @@NL@@<indent> with @@indent@@. Note this string munging is safe
+      # because inside whitespace, @@indent@@ is never legal after a newline
+      # (only after a comment, which needs a '#' after a newline).
+      # This might not work in the presence of '\' line continuations.
```

Comment:
Is this still true / okay? Is there a TODO here?

Reply:
Ah. I haven't tested, and I don't think that there are any tests that consider \ continuations. So I expect this to not work in general right now. 

The tokenizer will not emit anything for a '\', not even in the .src attribute of the tokens:

```
"this is a" + \
"continued line"
```

turns into

```
1,0-1,11:	STRING	'"this is a"'
1,12-1,13:	OP	'+'
2,0-2,16:	STRING	'"continued line"'
2,16-2,17:	NEWLINE	'\n'
3,0-3,1:	NL	'\n'
4,0-4,0:	ENDMARKER	''
```

So I think handling this properly might be infeasible?

Reply:
Interestingly, I think this means that `''.join([t.src for t in tokenize.generate_tokens(src)]) != src`. That pretty much guarantees that none of the usual guarantees pasta makes will hold in the presence of '\'.

Reply:
This is fixable if we want to add back the '\\\n' into the whitespace in token_generator. I think that might work, but I'm not sure it's worth doing.